test/drenv/__main__.py

```diff
@@ -37,23 +40,27 @@ def main():
         env = envfile.load(f, name_prefix=args.name_prefix)
 
     func = globals()[CMD_PREFIX + args.command]
-    func(env)
+
+    func(env, args)
 
 
-def cmd_start(env):
+def cmd_start(env, args):
     start = time.monotonic()
     logging.info("[%s] Starting environment", env["name"])
     # Delaying `minikube start` ensures cluster start order.
```

Comment:
The extra blank line is nice, but now the comment about delaying is far away from
the code and likely to become stale. Please keep the comment above the line using delay=1.